### ISSUE TITLE
SET_VALUE failed with ClientException

### ISSUE THREAD

**Issue by hvg1** [SukramJ/hahomematic#1027]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic)
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Since the last update of Home Assistant to 2023.4.4 I did 3 days ago, which also included the update of the custom component to 1.34.0, some of my blinds don't work anymore when triggered from Home Assistant (inside RaspberryMatic they still work).
Also, blinds using Homematic IP still work, those with Homematic are the ones with the error although other Homematic devices still work properly.

### What version of HomematicIP (local) has the issue?

1.34.0

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic HA-Addon

### Which version of your homematic backend are you running?

3.69.6.20230407

### What hardware are you running for your system?

Proxmox

### Which config details do you use

- [X] Username and password
- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [X] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

### Anything in the logs that might be useful for us?

```txt
Logger: hahomematic.client
Source: custom_components/homematicip_local/cover.py:151
Integration: Homematic(IP) Local ([documentation](https://github.com/danielperna84/custom_homematic), [issues](https://github.com/danielperna84/hahomematic/issues))
First occurred: April 13, 2023 at 20:56:59 (8 occurrences)
Last logged: April 15, 2023 at 20:00:29

    SET_VALUE failed with ClientException [(<Fault -1: 'Failure'>,)]: PEQ0098290:1, LEVEL_COMBINED, 0x0,0xc8
    SET_VALUE failed with ClientException [(<Fault -1: 'Failure'>,)]: PEQ0098256:1, LEVEL_COMBINED, 0x0,0xc8
    SET_VALUE failed with ClientException [(<Fault -1: 'Failure'>,)]: PEQ0097297:1, LEVEL_COMBINED, 0x0,0x0
    SET_VALUE failed with ClientException [(<Fault -1: 'Failure'>,)]: PEQ0098290:1, LEVEL_COMBINED, 0x0,0x0
    SET_VALUE failed with ClientException [(<Fault -1: 'Failure'>,)]: PEQ0098256:1, LEVEL_COMBINED, 0x0,0x0
```

### Additional information

RaspberryMatic Log:
```
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0x0,0x0" LogicalToPhysical failed
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0x0,0x0" LogicalToPhysical failed
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 15 20:00:29 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0x0,0x0" LogicalToPhysical failed
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0xc8,0x0" LogicalToPhysical failed
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0xc8,0x0" LogicalToPhysical failed
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::fromHexValue(): Cannot parse hex-string value (wrong size).
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSTypeConversionHexStringToByteArray::LogicalToPhysical(): Conversion failed.
Apr 16 05:59:57 de838cd8-raspberrymatic user.err rfd: HSSParameter::SetValue() "0xc8,0x0" LogicalToPhysical failed
```

**Comment by SukramJ** [comment]
Thanks for reporting.
should be fixed with Version 1.34.1

**Comment by hvg1** [comment]
> Thanks for reporting. should be fixed with Version 1.34.1

I can verify that at least the manual trigger is working again with 1.34.1, will report back as soon as some automations started triggers.
Thanks!

**Comment by hvg1** [comment]
Also works with automations again.
